modules/core/shared/src/test/scala/freestyle/free/free.scala

```diff
@@ -80,11 +80,15 @@ class freeTests extends WordSpec with Matchers {
     }
 
     "a trait with some concrete non-FS members" in {
-      """@free trait X {
+      @free trait X {
         def x: FS[Int]
         def y: Int = 5
         val z: Int = 6
-      }""" should compile
+      }
+      implicit object IdX extends X.Handler[Id]{
+        def x: Int = 31
+      }
+      0 shouldEqual 0
```

Comment:
😄

Reply:
I just want it to compile the code... but maybe it can be better.